build.gradle

```diff
@@ -21,10 +21,11 @@ checkstyle {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'    
 	runtimeOnly 'mysql:mysql-connector-java'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
-task checkstyle {
+task check {
```

Comment:
Remember to update the README instructions on how to run this task locally. `./gradlew checkstyle` is now `./gradlew check`